build/buildRunTimeImages.sh

```diff
@@ -15,6 +15,140 @@ source $REPO_DIR/build/__functions.sh
 source $REPO_DIR/build/__sdkStorageConstants.sh
 source $REPO_DIR/build/__stagingRuntimeConstants.sh
 
+# Load platform versions for runtimes
+source $REPO_DIR/build/__dotNetCoreRunTimeVersions.sh
+source $REPO_DIR/build/__nodeVersions.sh
+source $REPO_DIR/build/__phpVersions.sh
+source $REPO_DIR/build/__pythonVersions.sh
+source $REPO_DIR/build/__rubyVersions.sh
+
+# Get the specific platform version used for each runtime image to assist with future patching
```

Comment:
nit: we can shorten `getRuntimeTagVersion` a little:
```

#!/bin/bash

declare -A versionMappings

# Define version mappings
versionMappings["dotnetcore"]="8.0:NET_CORE_APP_80 7.0:NET_CORE_APP_70 6.0:NET_CORE_APP_60 5.0:NET_CORE_APP_50 3.1:NET_CORE_APP_31 3.0:NET_CORE_APP_30"
versionMappings["node"]="18:NODE18_VERSION 16:NODE16_VERSION 14:NODE14_VERSION"
versionMappings["php"]="8.2:PHP82_VERSION 8.1:PHP81_VERSION 8.0:PHP80_VERSION 7.4:PHP74_VERSION"
versionMappings["python"]="3.11:PYTHON311_VERSION 3.10:PYTHON310_VERSION 3.9:PYTHON39_VERSION 3.8:PYTHON38_VERSION 3.7:PYTHON37_VERSION"
versionMappings["ruby"]="2.7:RUBY27_VERSION 2.6:RUBY26_VERSION 2.5:RUBY25_VERSION"

# Function to get full runtime tag version
function getRuntimeTagVersion() {
    local PLATFORM_NAME="$1"
    local PLATFORM_VERSION="$2"
    
    # Check if the platform name is valid
    if [ -z "${versionMappings["$PLATFORM_NAME"]}" ]; then
        echo "Unable to retrieve version from the provided runtime platform name '$PLATFORM_NAME'"
        return 1
    fi

    # Use associative array for version mappings
    local versions="${versionMappings["$PLATFORM_NAME"]}"
    for versionMapping in $versions; do
        IFS=':' read -ra mapping <<< "$versionMapping"
        key="${mapping[0]}"
        value="${mapping[1]}"
        if [ "$PLATFORM_VERSION" == "$key" ]; then
            FULL_RUNTIME_TAG_VERSION="${!value}"
            return 0
        fi
    done

    # If version not found in mappings, use the provided version
    FULL_RUNTIME_TAG_VERSION="$PLATFORM_VERSION"
    return 0
}
```

Reply:
```
# Example usage:
PLATFORM_NAME="dotnetcore"
PLATFORM_VERSION="7.0"
getRuntimeTagVersion "$PLATFORM_NAME" "$PLATFORM_VERSION"
echo "Full Runtime Tag Version: $FULL_RUNTIME_TAG_VERSION"
```

Reply:
Gentle reminder regarding using code above.

Reply:
This is a good refactoring comment -- I decided not to make the change for this PR since Paul left a comment below that may change how we're reading versions, so we may scrap this logic in the future, but I should be able to blend your feedback with a future refactor involving Paul's proposal.